randovania/games/cave_story/generator/hint_distributor.py

```diff
@@ -25,7 +25,7 @@ def num_joke_hints(self) -> int:
         return 0
 
     @override
-    async def get_specific_pickup_precision_pairs(self) -> dict[NodeIdentifier, PrecisionPair]:
+    async def get_specific_location_precision_pairs(self) -> dict[NodeIdentifier, PrecisionPair]:
```

Comment:
🙃

Reply:
oh whoops that's a leftover from the changes i started working on before realizing i wanted this